### ISSUE TITLE
Getting seduced through cloak

### ISSUE THREAD

**Issue by Jiaku** [dalaranwow/dalaran-wow#5062]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If you are getting seduced by warlock's pet and you trinket it as a rogue and use cloak of shadows right away you will get seduced through cloak.
**Expected behaviour**: Tell us what should happen instead
you are not getting seduced through cloak
**Steps to reproduce the problem**:

1. Duel a warlock, ask to seduce you (warlock must have talents for fast seduce)
2. Use trinket and use cloak
3. you will get seduced through cloak 100%

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Bajeru** [comment]
Hi Jiaku, i've had same problems. But Cloak of Shadows is not 100% but 90%. Tho, i agree with your saying, it doesnt seem to work properly when you use it after you have used the PvP trinket. But there is still 10% chance it would go through, but you get charmed 9/10 times still where it should be more like 1/10 times.

**Comment by Jiaku** [comment]
I know it is not 100% resists but i did some tests and it happend 5 times in a row.

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Jiaku.
Issue confirmed on PTR and no it's not the 10% chance to hit though cloak.
Spell: https://wowgaming.altervista.org/aowow/?spell=6358 hits the rogue though spell https://wowgaming.altervista.org/aowow/?spell=31224 every single time.

**Comment by Jiaku** [comment]
Any news?

**Comment by Jiaku** [comment]
Any news, thought, ideas, workarounds?

**Comment by robinsch** [comment]
@Lothloryen @TeridanDW I can't confirm the issue on my local PTR. It seems like that there is some kind of DB change to that?

It missed 12/12 times for me.

**Comment by TeridanDW** [comment]
Just tested the issue again and got the same results.
Try to start a duel, rogue stays still, summon pet and order an attack, cast cloack while pet is casting seduce and notice you got hit always (for me that is on PTR).

**Comment by robinsch** [comment]
Yeah, working fine for me.

**Comment by Jiaku** [comment]
Speaking of cloak, if you get hit with coil let's say while you have cloak up, warlock can apply ANY spell he wants through cloak.

**Comment by Jiaku** [comment]
I have noticed one thing, if you pressed cloak while pet is casting seduction you will get charmed, if you press cloak before cast you won't get charmed since pet wont able to cast seduction. Not so sure on this but pet should be able to cast seduction regardless if you have cloak up or not and if it does it should screw diminishing return. With that said, i guess succy checks if it possible to lend seduction at the start of cast instead of at the end, that's why seduction getting through cloak. Altho it is just my guess.

**Comment by Jiaku** [comment]
Any news?

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the feedback - I have been doing several tests and got similar results than @robinsch 's. I only got charmed 2 from 30 times, casting it either before starting the Seduction cast or while  it was being cast.

Maybe I did not perform the correct steps so, any ideas or feedback about how to reproduce it would be more than welcome since currently it worked just fine for me.

**Comment by Jiaku** [comment]
@Lothloryen  TeridanDW managed to reproduce the issue

**Comment by Jiaku** [comment]
Still getting seduced through cloak. The desease from ice touch is always applies through cloak(not slow,just a desease). Hunter's sting applies through cloak and hunter's mark. And huge pile of other spells goes through cloak like paladin's HoJ not even close to 1/10. I highly suggest to do some testing on cloak of shadows.

**Comment by Jiaku** [comment]
So, is it getting fixed?

**Comment by Jiaku** [comment]
https://www.youtube.com/watch?v=OztBx_JXivo&feature=youtu.be
